<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NamingDialog</class>
 <widget class="QDialog" name="NamingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>174</width>
    <height>88</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Naming Options - Lightscreen</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="container" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_11">
      <property name="margin">
       <number>9</number>
      </property>
      <item>
       <widget class="QStackedWidget" name="stack">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="numericStackPage">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QGridLayout" name="namePageLayout">
          <property name="margin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="leadingZerosLabel">
            <property name="text">
             <string>Leading zeros:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="leadingZerosSpinBox">
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::PlusMinus</enum>
            </property>
            <property name="suffix">
             <string notr="true"/>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>25</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="dateStackPage">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="horizontalSpacing">
           <number>0</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="dateFormatLabel">
            <property name="text">
             <string>Date Format:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>1</width>
              <height>1</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="dateHelpLabel">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">margin-left: 4px</string>
            </property>
            <property name="text">
             <string notr="true"> &lt;a href=&quot;http://lightscreen.sourceforge.net/help/date&quot;&gt;[?]&lt;/a&gt;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QComboBox" name="dateFormatComboBox">
            <property name="editable">
             <bool>true</bool>
            </property>
            <item>
             <property name="text">
              <string notr="true">yyyy-MM-dd</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">yyyy-MM-dd HH.mm.ss</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="timestampStackPage">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="flipNamingCheckBox">
        <property name="text">
         <string>Flip naming.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NamingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NamingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
